install_config/storage_examples/privileged_pod_storage.adoc

```diff
@@ -112,14 +112,14 @@ $ oc create -f gluster-pvc.yaml -n <project_name>
 [source,yaml]
 ----
 apiVersion: v1
-id: gluster-nginx-pvc
+id: gluster-S3-pvc
 kind: Pod
 metadata:
-  name: gluster-nginx-priv
+  name: gluster-S3-priv
 spec:
   containers:
-    - name: gluster-nginx-priv
-      image: fedora/nginx
+    - name: gluster-S3-priv
+      image: fedora/S3
```

Comment:
not sure what the mechanism of this image are

Reply:
@mdshuai Any thoughts on this change ^^

Reply:
I don't think we have an image with this name.

\# docker pull fedora/nginx
Using default tag: latest
Trying to pull repository registry.access.redhat.com/fedora/nginx ... 
Trying to pull repository registry.reg-aws.openshift.com:443/fedora/nginx ... 
Pulling repository registry.reg-aws.openshift.com:443/fedora/nginx
Trying to pull repository docker.io/fedora/nginx ... 
latest: Pulling from docker.io/fedora/nginx
......

\# docker pull fedora/S3
Error parsing reference: "fedora/S3" is not a valid repository/tag

\# docker pull fedora/s3
Using default tag: latest
Trying to pull repository registry.access.redhat.com/fedora/s3 ... 
Trying to pull repository registry.reg-aws.openshift.com:443/fedora/s3 ... 
Pulling repository registry.reg-aws.openshift.com:443/fedora/s3
Trying to pull repository docker.io/fedora/s3 ... 
Pulling repository docker.io/fedora/s3
Error: image fedora/s3:latest not found

Reply:
@liangxia @sjenning 
Can we leave the original `gluster-nginx-priv` here as the section specifically requires "A user with the cluster-admin role binding." Does this cover the _super-user privileges_ that was the original concern of the submitter, @nak3

Reply:
@sjenning @liangxia @nak3 ^^

Reply:
I'm OK to revert this specific one.